### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by josephl310**
Update README instructions to get course/branch id and also fix part on timeout limits

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -100,15 +100,27 @@ file within that directory contains a refresh token for the user who authorized
...
+    
+The response will be a JSON object containing an ``id`` field with the value::
+
+    iRl53_BWEeW4_wr--Yv6Aw
+    
+3. If the grader that you're looking to upload/update is not in the `Original Version` of the course, you'll most likely need to get a branch id. To this, you'll need the ``courseId`` and the ``versionName``. Make the following request::
+
+    https://api.coursera.org/api/authoringBranchProperties.v1?q=course&courseId={course_id}&fields=properties
+    
+Search for your version name (it'll be in the ``properties.name`` field), and find the associated ``id`` (it should look something like: ``authoringBranch~xxxxxxxxxxxxxxxxxxxxxx``.
```
Comment (sashi-coursera) [inline README.rst:123]: Will it be always `authoringBranch~xxx` for base versions as well?

**Review by sashi-coursera — APPROVED** [review]
Thanks Joseph.

File: README.rst (hunk 1 above)
Comment (shreve) [reply README.rst:123]: I believe so. That's what I heard from the person I spoke with at Coursera, and the data I've seen bears this out.

Comment (josephl310) [reply README.rst:123]: The original version won't have the `authoringBranch` prefix, but all others will

Comment (sashi-coursera) [reply README.rst:123]: OK, in that case, it might be helpful to provide that distinction here.

Comment (josephl310) [inline README.rst:123]:
```suggestion
Search for your version name (it'll be in the ``properties.name`` field), and find the associated ``id`` (it should look something like: ``authoringBranch~xxxxxxxxxxxxxxxxxxxxxx``. (Please note that this form of branch id is only applicable for non-original versions of the course. For the original version, the id will be a twenty-two character long string with no prefix).
```

**Pull request merged**